MonoGame.Framework/Windows/WinFormsGameForm.cs

```diff
@@ -32,6 +32,7 @@ public static System.Drawing.Point GetPointerLocation(this Message msg)
     internal class WinFormsGameForm : Form
     {
         GameWindow _window;
+        public const int WM_MOUSEHWHEEL = 0x020E;
```

Comment:
I'm not a fan of the constant, but I was trying to use the same macro name that Microsoft uses.  It's not that I don't spell out WHEEL, it's that there is an H before it that spells HWHEEL.  I know that I did a double take just looking for the proper message macro for a horizontal mouse wheel.
https://msdn.microsoft.com/en-us/library/windows/desktop/ms645614(v=vs.85).aspx